### PR TITLE
Fixes #17793: Importing technique leads to a blocked editor

### PR DESCRIPTION
**Pull request by ElaadF**
https://issues.rudder.io/issues/17793

### CODE REVIEW

File: builder/js/ncf.js [hunk 1]
```diff
@@ -1066,7 +1111,7 @@ $scope.onImportFileChange = function (fileEl) {
     return { "method_calls" : []
     , "name"         : ""
     , "description"  : ""
-    , "version"      : "1.0"
```
Comment (VinceMacBuche) [inline builder/js/ncf.js:1069]: it's not the same version ;) should still be 1.0! here it is the version of the technique not the import version

File: builder/js/ncf.js [hunk 2]
```diff
@@ -1337,7 +1382,7 @@ $scope.onImportFileChange = function (fileEl) {
       ncfTechnique = data.data.techniques.technique;
 
       // Transform back ncfTechnique to UITechnique, that will make it ok
-      var savedTechnique = toTechUI(ncfTechnique);
+      var savedTechnique = toTechUI(ncfTechnique, ncfTechnique['version']);
```
Comment (VinceMacBuche) [inline builder/js/ncf.js:1385]: you can say it's 2.0 here.

**Comment by ElaadF** [comment]
Commit modified

File: builder/js/ncf.js [hunk 3]
```diff
@@ -1337,7 +1382,7 @@ $scope.onImportFileChange = function (fileEl) {
       ncfTechnique = data.data.techniques.technique;
 
       // Transform back ncfTechnique to UITechnique, that will make it ok
-      var savedTechnique = toTechUI(ncfTechnique);
+      var savedTechnique = toTechUI(ncfTechnique, 2.0);
```
Comment (VinceMacBuche) [inline builder/js/ncf.js:1385]: should be a string

Comment (VinceMacBuche) [reply builder/js/ncf.js:1385]: In fact ... it does not work with an int, and you could simplify with 1 or 2

Comment (VinceMacBuche) [reply builder/js/ncf.js:1385]: ok I'm merging then

**Pull request merged**